google/cloud/opentelemetry/README.md

```diff
@@ -90,10 +94,12 @@ int main(int argc, char* argv[]) {
 
 - If you want to learn more about the [OpenTelemetry] for C++ SDKs, consult
```

Comment:
In other libraries we removed the `Build and Install` section in favor of a link to https://cloud.google.com/cpp/docs/setup/